test/integration_test/specs/mysql-clone-disallowed-ns/appclone.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: stork.libopenstorage.org/v1alpha1
+kind: ApplicationClone
+metadata:
+  name: clonemysql
```

Comment:
nit: clone-mysql
You may change everywhere if you want.

Reply:
Updated